### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by AswathyVNair**
Preview: https://deploy-preview-5--bakehere.netlify.app/pages/login.html

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            </div>
+        </div>
+        <div class = "btn-nav">
+            <button class="btn-login">
+              <a href = "/" >SignUp</a> 
+            </button>
+            <button class ="btn-icon">
+                <a href="/"><i class="bi bi-heart-fill"></i></a>
+            </button>
+            <button class ="btn-icon">
```
Comment (itsmebipul00) [inline pages/login.html:33]: you could make a look like a button

**Review by itsmebipul00 — APPROVED** [review]
nice work

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.login-pg-maincontent{
+    display: flex;
+    justify-content: center;
+    height: 100vh;
+}
+.login-container{
+    border: 2px solid var(--green);
```
Comment (bhushannaikaroskar) [inline css/login.css]: See you code is well written just use consistent spacing between different classes for better readability

File: css/login.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+.login-container .btn-login:hover{
+    background-color: var(--dark-green);
+}
+
+.login-container a{
+    color: var(--green);
+}
+.login-container a:hover{
+    color: var(--dark-green);
```
Comment (bhushannaikaroskar) [inline css/login.css]: Add proper spacing here as well

**Review by bhushannaikaroskar — APPROVED** [review]
Your code is clean and well formatted. All I can see as of now is that you must use consistent spacing throughout. I would suggest you to use vscode plugin prettier to auto format spacing. Rest I approve your code.

File: css/login.css (hunk 2 above)
Comment (AswathyVNair) [reply css/login.css]: Thanks for the suggestion. I have made changes.

**Pull request merged**